hal/src/rtl872x/inet_hal_compat.cpp

```diff
@@ -34,3 +34,16 @@ uint32_t HAL_NET_SetNetWatchDog(uint32_t) {
 
 void HAL_NET_SetCallbacks(const HAL_NET_Callbacks* callbacks, void* reserved) {
 }
+
+int inet_gethostbyname_wlan(const char* hostname, uint16_t hostnameLen, HAL_IPAddress* out_ip_addr, network_interface_t nif, void* reserved)
```

Comment:
Are these stub functions an acceptable workaround for the conflicts with dynalib now that this platform has both cellular and wifi dynalib tables?